drivers/fuel_gauge/max17048/emul_max17048.c

```diff
@@ -69,13 +69,10 @@ static int max17048_emul_transfer_i2c(const struct emul *target, struct i2c_msg
 				       int num_msgs, int addr)
 {
 	/* Largely copied from emul_bmi160.c */
-	struct max17048_emul_data *data;
```

Comment:
I think this change was accidentally included?

Reply:
Done